_data/simple-icons.json

```
@@ -12024,6 +12024,12 @@
             "hex": "FF6633",
             "source": "https://portswigger.net"
         },
+        {
+            "title": "Posit",
+            "hex": "447099",
+            "source": "https://docs.posit.co",
+            "guidelines": "https://posit.co/about/trademark-guidelines/"
```

```suggestion
            "guidelines": "https://posit.co/about/trademark-guidelines"
```

Gotta save those bytes where we can! Removing the trailing slash still works.